File: week18/김고은/블로그2/20365.js
// 20M
// 문자열 + 단순 반복문 
const fs = require('fs');
const filePath = process.platform === 'linux'? '/dev/stdin' : __dirname + '/input.txt';
const [N, arr] = fs.readFileSync(filePath).toString().trim().split('\n')

let colors = [];
colors = arr.split('');

let paintBlueAfter = 1;
let paintRedAfter = 1;

let splitedByRed = arr.split('R');
let splitedByBlue = arr.split('B');


for(let i = 0; i < splitedByRed.length; i++){
    if(splitedByRed[i] === '') continue;
    else paintBlueAfter++;
}

for(let i = 0; i < splitedByBlue.length; i++){
    if(splitedByBlue[i] === '') continue;
    else paintRedAfter++;
}

console.log(paintBlueAfter > paintRedAfter? paintRedAfter: paintBlueAfter )

File: week18/김고은/블로그2/20365.py
import sys

input = sys.stdin.readline

N = int(input().strip())
arr = input().strip()

paint_blue_after = 1
paint_red_after = 1

splited_by_red = arr.split('R')
splited_by_blue = arr.split('B')

for i in range(len(splited_by_red)):
    if splited_by_red[i] == '':
        continue
    else:
        paint_blue_after += 1

for i in range(len(splited_by_blue)):
    if splited_by_blue[i] == '':
        continue
    else:
        paint_red_after += 1

print(min(paint_blue_after, paint_red_after))